Add caveat about rotating the non-desktop environment (#4163)
raspberrypi/documentation: documentation/asciidoc/accessories/touch-display-2/about.adoc

video=DSI-1:720x1280@60,rotate=<rotation-value>
----

> NOTE: This `rotate=` setting only rotates the text-mode console; any applications that write directly to DRM (e.g. libcamera or cvlc) won't be rotated, and will instead need to use their own rotation options (if available).

Reviewer: @lurch In answer to your question, I don't think you need "NOTE" at all. Just go straight into the sentence. Also, replace "e.g." with "for example" or "such as".

Author: Fixed and fixed

> This `rotate=` setting only rotates the text-mode console; any applications that write directly to DRM (for example libcamera or cvlc) won't be rotated, and will instead need to use their own rotation options (if available).

Author: ```suggestion
This `rotate=` setting only rotates the text-mode console; any applications that write directly to DRM (such as `cvlc` or the libcamera apps) won't be rotated, and will instead need to use their own rotation options (if available).
```
